secrets/dataplane-observability-config.yaml.sample

```diff
@@ -0,0 +1,40 @@
+---
+# [required]
+remote_write_configuration:
+  # [required] V2 URL endpoint of the Prometheus instance to be used by the data planes
+  # to send metrics. The Prometheus instance has to offer a Remote Write endpoint.
+  # If oidc_configuration is set that endpoint has to be offered under the
+  # /api/v1/metrics/<tenant>/api/v1/receive" URL path, where <tenant> is defined by the
+  # --observability-red-hat-sso-tenant Fleet Manager flag.
+  # If oidc_coniguration is set that endpoint has to be offered under the
+  # "/" URL path.
```

Comment:
I find this description confusing. Does this mean that the endpoint has to be offered under both `/` and `/api/v1/metrics/<tenant>/api/v1/receive`?

Reply:
I believe a `not` is missing in the second part.

Reply:
Yes. a `not` is missing in the second part :+1: